app/js/controller/list_controller.js

```diff
@@ -11,16 +11,30 @@ export default class ListController extends Controller {
     this.webServer = new WebServer();
   }
 
-  main() {    
-    this.createList();
+  main() {
+    this.model.getAllApps().then(allApps => {
```

Comment:
nit: I usually wrap the function arguments in `(` `)` for these fat-arrow functions if the function spans multiple lines.

e.g.:
```
    this.model.getAllApps().then((allApps) => {
```